### ISSUE TITLE
Rendering locally formatted numbers similar to normal elements 

### ISSUE THREAD

**Issue by larsjuvik** [almond-bongbong/react-slot-counter#42]
Hi, thanks for writing this library - it's really good! 😄👍

I'm trying to display a number for Norwegian locale in `<SlotCounter />` in the same way it would be for a `<p>`-element. The difference can be seen in the picture below.

<img width="245" alt="Screenshot 2024-05-22 at 00 41 02" src="https://github.com/almond-bongbong/react-slot-counter/assets/77640590/3d3b14b6-d17f-4cb0-8806-360166e52d82">

When the webpage refreshes, it looks like they are the same length for the first frame. Then `<SlotCounter />`  renders numbers longer than the bottom number. I can get them kind of similar when using negative margins in `charClassName` - but the numbers look a bit "squished" together, and the gaps are still wide. The space between the numbers are not recognised as a separator, so I don't think setting `separatorClassName` will help in this case.

I'm wondering If there is an easy and good way to do this? Any help would be much appreciated! 😄 

The example function if interested:
```js
export default function Home() {
  const value = 123456789;
  const [valueNorwegianLocale] = useState(value.toLocaleString("nb-NO"));

  return (
    <>
      <p>react-slot-counter</p>
      <SlotCounter autoAnimationStart={false} value={valueNorwegianLocale} />

      <p className="mt-5">paragraph</p>
      <p>{valueNorwegianLocale}</p>
    </>
  );
}
```

**Comment by larsjuvik** [comment]
Hi, and thanks! If it's desirable to add this as a separator, I can try to add support for this.

I will look at the codebase a bit more later, but maybe it's as easy as including the character in this: `const SEPARATOR = [',', '.', ' '];`?

**Comment by almond-bongbong** [comment]
We have addressed this issue in version 2.3.3. The SlotCounter component now correctly handles non-breaking spaces (NBSP) as separators. This update ensures that locally formatted numbers render consistently, similar to normal HTML elements. Thank you for bringing this to our attention and helping us improve the library. If you encounter any further issues or have additional feedback, please let us know. 👍

**Comment by larsjuvik** [comment]
Thank you so much, this works perfectly now! 👍
